### ISSUE TITLE
Problems with interpretation of training results

### ISSUE THREAD

**Issue by MakkaPakka-sc** [megvii-research/mdistiller#61]
![Snipaste_2024-03-04_15-34-44](https://github.com/megvii-research/mdistiller/assets/69374394/cf67041b-7538-4b99-ab91-0b3e7f541e5d)
Thank you very much for your research content. I changed the teacher model to GFNet, and the student model used MobileNet V2. I got this result. What is the reason for the mutation when the step is 300? In addition, which indicator represents the test acc of the student model and which represents the acc of the teacher model. Is the calculation of FLOPs integrated in the framework?

**Comment by MakkaPakka-sc** [comment]
@Zzzzz1 Can you answer my question? Thank you very much

**Comment by Zzzzz1** [comment]
1. The reason is that the learning rate changes. 2.test_acc represents the acc of the student, the teacher's acc is not plotted. 3. not integrated.

**Comment by MakkaPakka-sc** [comment]
Thank you for your answer. What may be the reason for the unchanged acc on the later test set?
